<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN"
   "http://www.w3.org/TR/html4/strict.dtd">
<html lang="en">
<head>
    <meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
    <meta name="robots" content="noindex,nofollow,noarchive">
<meta http-equiv="Refresh" content="86400">
    <link rel="stylesheet" href="style.css" type="text/css">
    <script>
function displayAlert(alertStr)
{
alert(alertStr);
}
</script>
<script src="saveDataGist.js"></script>
    <title>rmflight feeds</title>
</head>
<body id="rawdog">
<div id="header">
<h1>rawdog</h1>
</div>
<div id="buttons">
<p><input type="button" value="token" OnClick="askToken()"></p>
<p><input type="button" value="initial" OnClick="getInitial()"></p>
<p><input type="button" value="patch" OnClick="sendPatch()"></p>
</div>
<div id="items">
<div class="feeddisplay">
<h3 class="feedtitle"><a id="0"><a href="http://www.biodatamining.org/">BioData Mining - Latest Articles</a></a></h3>
<a href="#1">next</a>
<ol class="feedarticles">
<div class="item feed-0320d9ef feed-biodatamininglatestarticles" id="item-57f1d0aa">
<p class="itemheader">
<span class="itemtitle"><a href="http://www.biodatamining.org/content/6/1/7">Optimization and visualization of the edge weights in optimal assignment methods for virtual screening</a></span>
<span class="itemsave"><input type="button" onclick="sendPatch('http://www.biodatamining.org/content/6/1/7')" value="Save"></span>
<span class="itemfrom">[<a href="http://www.biodatamining.org/">BioData Mining - Latest Articles</a>]</span>
</p>
<div class="itemdescription">
<p>The optimization and visualization of edge weights in optimal assignment methods is a valuable approach for ligandbased virtual screening experiments and provides a useful tool for drug discovery.</p>
</div>
</div>
<div class="item feed-0320d9ef feed-biodatamininglatestarticles" id="item-e2ea74a2">
<p class="itemheader">
<span class="itemtitle"><a href="http://www.biodatamining.org/content/6/1/8">Prediction of Drosophila melanogaster gene function using Support Vector Machines</a></span>
<span class="itemsave"><input type="button" onclick="sendPatch('http://www.biodatamining.org/content/6/1/8')" value="Save"></span>
<span class="itemfrom">[<a href="http://www.biodatamining.org/">BioData Mining - Latest Articles</a>]</span>
</p>
<div class="itemdescription">
<p>Background:
While the genomes of hundreds of organisms have been sequenced and good approaches exist for finding protein encoding genes, an important remaining challenge is predicting the functions of the large fraction of genes for which there is no annotation. Large gene expression datasets from microarray experiments already exist and many of these can be used to help assign potential functions to these genes. We have applied Support Vector Machines (SVM), a sigmoid fitting function and a stratified cross&#8208;validation approach to analyze a large microarray experiment dataset from Drosophila melanogaster in order to predict possible functions for previously un&#8208;annotated genes. A total of approximately 5043 different genes, or about one&#8208;third of the predicted genes in the D. melanogaster genome, are represented in the dataset and 1854 (or 37%) of these genes are un&#8208;annotated.
Results:
39 Gene Ontology Biological Process (GO&#8208;BP) categories were found with precision value equal or larger than 0.75, when recall was fixed at the 0.4 level. For two of those categories, we have provided additional support for assigning given genes to the category by showing that the majority of transcripts for the genes belonging in a given category have a similar localization pattern during embryogenesis. Additionally, by assessing the predictions using a confidence score, we have been able to provide a putative GO&#8208;BP term for 1422 previously un&#8208;annotated genes or about 77% of the un&#8208;annotated genes represented on the microarray and about 19% of all of the un&#8208;annotated genes in the D. melanogaster genome.
Conclusions:
Our study successfully employs a number of SVM classifiers, accompanied by detailed calibration and validation techniques, to generate a number of predictions for new annotations for D. melanogaster genes. The applied probabilistic analysis to SVM output improves the interpretability of the prediction results and the objectivity of the validation procedure.
</div>
</div>
<div class="item feed-0320d9ef feed-biodatamininglatestarticles" id="item-a01a105d">
<p class="itemheader">
<span class="itemtitle"><a href="http://www.biodatamining.org/content/6/1/9">A robustness study of parametric and non-parametric tests in model-based multifactor dimensionality reduction for epistasis detection</a></span>
<span class="itemsave"><input type="button" onclick="sendPatch('http://www.biodatamining.org/content/6/1/9')" value="Save"></span>
<span class="itemfrom">[<a href="http://www.biodatamining.org/">BioData Mining - Latest Articles</a>]</span>
</p>
<div class="itemdescription">
<p>Background:
Applying a statistical method implies identifying underlying (model) assumptions and checking their validity in the particular context. One of these contexts is association modeling for epistasis detection. Here, depending on the technique used, violation of model assumptions may result in increased type I error, power loss, or biased parameter estimates. Remedial measures for violated underlying conditions or assumptions include data transformation or selecting a more relaxed modeling or testing strategy. Model-Based Multifactor Dimensionality Reduction (MB-MDR) for epistasis detection relies on association testing between a trait and a factor consisting of multilocus genotype information. For quantitative traits, the framework is essentially Analysis of Variance (ANOVA) that decomposes the variability in the trait amongst the different factors. In this study, we assess through simulations, the cumulative effect of deviations from normality and homoscedasticity on the overall performance of quantitative Model-Based Multifactor Dimensionality Reduction (MB-MDR) to detect 2-locus epistasis signals in the absence of main effects.MethodologyOur simulation study focuses on pure epistasis models with varying degrees of genetic influence on a quantitative trait. Conditional on a multilocus genotype, we consider quantitative trait distributions that are normal, chi-square or Student&#8217;s t with constant or non-constant phenotypic variances. All data are analyzed with MB-MDR using the built-in Student&#8217;s t-test for association, as well as a novel MB-MDR implementation based on Welch&#8217;s t-test. Traits are either left untransformed or are transformed into new traits via logarithmic, standardization or rank-based transformations, prior to MB-MDR modeling.
Results:
Our simulation results show that MB-MDR controls type I error and false positive rates irrespective of the association test considered. Empirically-based MB-MDR power estimates for MB-MDR with Welch&#8217;s t-tests are generally lower than those for MB-MDR with Student&#8217;s t-tests. Trait transformations involving ranks tend to lead to increased power compared to the other considered data transformations.
Conclusions:
When performing MB-MDR screening for gene-gene interactions with quantitative traits, we recommend to first rank-transform traits to normality and then to apply MB-MDR modeling with Student&#8217;s t-tests as internal tests for association.
</div>
</div>
<div class="item feed-0320d9ef feed-biodatamininglatestarticles" id="item-b8dd3863">
<p class="itemheader">
<span class="itemtitle"><a href="http://www.biodatamining.org/content/6/1/10">The limits of p-values for biological data mining</a></span>
<span class="itemsave"><input type="button" onclick="sendPatch('http://www.biodatamining.org/content/6/1/10')" value="Save"></span>
<span class="itemfrom">[<a href="http://www.biodatamining.org/">BioData Mining - Latest Articles</a>]</span>
</p>
<div class="itemdescription">
<p>This insightful Editorial discusses the applications and interpretations of p-values underlying analytical approaches in data mining and how they can be more complex than one might assume.</p>
</div>
</div>
<div class="item feed-0320d9ef feed-biodatamininglatestarticles" id="item-86378b50">
<p class="itemheader">
<span class="itemtitle"><a href="http://www.biodatamining.org/content/6/1/11">MicroRNA-mediated regulation of target genes in several brain regions is correlated to both microRNA-targeting-specific promoter methylation and differential microRNA expression</a></span>
<span class="itemsave"><input type="button" onclick="sendPatch('http://www.biodatamining.org/content/6/1/11')" value="Save"></span>
<span class="itemfrom">[<a href="http://www.biodatamining.org/">BioData Mining - Latest Articles</a>]</span>
</p>
<div class="itemdescription">
<p>Background:
Public domain databases nowadays provide multiple layers of genome-wide data e.g., promoter methylation, mRNA expression, and miRNA expression and should enable integrative modeling of the mechanisms of regulation of gene expression. However, researches along this line were not frequently executed.
Results:
Here, the public domain dataset of mRNA expression, microRNA (miRNA) expression and promoter methylation patterns in four regions, the frontal cortex, temporal cortex, pons and cerebellum, of human brain were sourced from the National Center for Biotechnology Informations gene expression omnibus, and reanalyzed computationally. A large number of miRNA-mediated regulation of target genes and miRNA-targeting-specific promoter methylation were identified in the six pairwise comparisons among the four brain regions. The miRNA-mediated regulation of target genes was found to be highly correlated with one or both of miRNA-targeting-specific promoter methylation and differential miRNA expression. Genes enriched for Kyoto Encyclopedia of Genes and Genomes (KEGG) pathways that were related to brain function and/or development were found among the target genes of miRNAs whose differential expression patterns were highly correlated with the miRNA-mediated regulation of their target genes.
Conclusions:
The combinatorial analysis of miRNA-mediated regulation of target genes, miRNA-targeting-specific promoter methylation and differential miRNA expression can help reveal the brain region-specific contributions of miRNAs to brain function and development.
</div>
</div>
<div class="item feed-0320d9ef feed-biodatamininglatestarticles" id="item-03175f19">
<p class="itemheader">
<span class="itemtitle"><a href="http://www.biodatamining.org/content/6/1/12">The disconnect between classical biostatistics and the biological data mining community</a></span>
<span class="itemsave"><input type="button" onclick="sendPatch('http://www.biodatamining.org/content/6/1/12')" value="Save"></span>
<span class="itemfrom">[<a href="http://www.biodatamining.org/">BioData Mining - Latest Articles</a>]</span>
</p>
<div class="itemdescription">
<p>Statistics departments and journals still strongly emphasize a very narrow range of topics and methods and techniques, all driven by a tiny handful of results, many dating from the 1930s. Those methods may well have been good and amazing and quite appropriate for the available computing, known mathematical facts, and data of their day. Hence the common list of assumptions: normal distributions and very small parametric models and linearity and independent features. But the usual claims for these anchoring assumptions are accurate--when precisely true--but more often just irrelevant: data is rarely normal, model misspecification is always at work, features are highly entangled with functionally mysterious interactions, and multiple scientifically plausible models may all fit the data equally well. We describe here machine learning as an alternative.
</div>
</div>
<div class="item feed-0320d9ef feed-biodatamininglatestarticles" id="item-137de6f5">
<p class="itemheader">
<span class="itemtitle"><a href="http://www.biodatamining.org/content/6/1/13">Unraveling genomic variation from next generation sequencing data</a></span>
<span class="itemsave"><input type="button" onclick="sendPatch('http://www.biodatamining.org/content/6/1/13')" value="Save"></span>
<span class="itemfrom">[<a href="http://www.biodatamining.org/">BioData Mining - Latest Articles</a>]</span>
</p>
<div class="itemdescription">
<p>Elucidating the content of a DNA sequence is critical to deeper understand and decode the genetic information for any biological system. As next generation sequencing (NGS) techniques have become cheaper and more advanced in throughput over time, great innovations and breakthrough conclusions have been generated in various biological areas. Few of these areas, which get shaped by the new technological advances, involve evolution of species, microbial mapping, population genetics, genome-wide association studies (GWAs), comparative genomics, variant analysis, gene expression, gene regulation, epigenetics and personalized medicine. While NGS techniques stand as key players in modern biological research, the analysis and the interpretation of the vast amount of data that gets produced is a not an easy or a trivial task and still remains a great challenge in the field of bioinformatics. Therefore, efficient tools to cope with information overload, tackle the high complexity and provide meaningful visualizations to make the knowledge extraction easier are essential. In this article, we briefly refer to the sequencing methodologies and the available equipment to serve these analyses and we describe the data formats of the files which get produced by them. We conclude with a thorough review of tools developed to efficiently store, analyze and visualize such data with emphasis in structural variation analysis and comparative genomics. We finally comment on their functionality, strengths and weaknesses and we discuss how future applications could further develop in this field.
</div>
</div>
<div class="item feed-0320d9ef feed-biodatamininglatestarticles" id="item-c2001cec">
<p class="itemheader">
<span class="itemtitle"><a href="http://www.biodatamining.org/content/6/1/14">The central role of biological data mining in connecting diverse disciplines</a></span>
<span class="itemsave"><input type="button" onclick="sendPatch('http://www.biodatamining.org/content/6/1/14')" value="Save"></span>
<span class="itemfrom">[<a href="http://www.biodatamining.org/">BioData Mining - Latest Articles</a>]</span>
</p>
<div class="itemdescription">
<p>Scientific research has always suffered from the compartmentalization of different disciplines. This historic phenomenon is beginning to erode in specific broader disciplines such as medicine where new interdisciplinary areas such as bioinformatics have worked to bring together biologists with computer scientists and mathematicians. This is an encouraging trend but has yet to percolate to more disparate disciplines where there is tremendous potential for scientific advances. We had the pleasure and honor of participating in a 2013 Indonesian-American Kavli Frontiers of Science Symposium organized and sponsored jointly by the United States National Academy of Sciences and the Indonesian Academy of Sciences. This invitation-only scientific conference took place in Bali, Indonesia and was attended by young or early career scientists (Kavli Fellows) doing cutting-edge research in extremely diverse disciplines. The goals of the conference were two-fold. The first goal of was to expose researchers to very broad research topics to promote new research ideas within their own field or to expand their research into new fields. The second goal was to stimulate collaboration between U.S. and Indonesian researchers to foster a global exchange of ideas.
</div>
</div>
<div class="item feed-0320d9ef feed-biodatamininglatestarticles" id="item-ed8546a7">
<p class="itemheader">
<span class="itemtitle"><a href="http://www.biodatamining.org/content/6/1/15">gff2sequence, a new user friendly tool for the generation of genomic sequences</a></span>
<span class="itemsave"><input type="button" onclick="sendPatch('http://www.biodatamining.org/content/6/1/15')" value="Save"></span>
<span class="itemfrom">[<a href="http://www.biodatamining.org/">BioData Mining - Latest Articles</a>]</span>
</p>
<div class="itemdescription">
<p>Background:
General Feature Format (GFF) files are used to store genome features such as genes, exons, introns, primary transcripts etc. Although many software packages (i.e. ab initio gene prediction programs) can annotate features by using such a standard, a small number of tools have been developed to extract the corresponding sequence information from the original genome. However the present tools do not execute either a quality control or a customizable filter of the annotated features is available.Findingsgff2sequence is a program that extracts nucleotide/protein sequences from a genomic multifasta by using the information provided by a general feature format file. While a graphical user interface makes this software very easy to use, a C++ algorithm allows high performance together with low hardware demand. The software also allows the extraction of the genic portions such as the untranslated and the coding sequences. Moreover a highly customizable quality control pipeline can be used to deal with anomalous splicing sites, incorrect open reading frames and not canonical characters within the retrieved sequences.
Conclusions:
gff2sequence is a user friendly program that allows the generation of highly customizable sequence datasets by processing a general feature format file. The presence of a wide range of quality filters makes this tool also suitable for refining the ab initio gene predictions.
</div>
</div>
</ol></div>

<div class="feeddisplay">
<h3 class="feedtitle"><a id="1"><a href="http://genome.cshlp.org">Genome Research current issue</a></a></h3>
<a href="#0">prev</a> | <a href="#2">next</a>
<ol class="feedarticles">
<div class="item feed-f637ce59 feed-genomeresearchcurrentissue" id="item-897ee66a">
<p class="itemheader">
<span class="itemtitle"><a href="http://genome.cshlp.org/cgi/content/full/23/10/1704?rss=1">Meta-analyses of studies of the human microbiota [RESEARCH]</a></span>
<span class="itemsave"><input type="button" onclick="sendPatch('http://genome.cshlp.org/cgi/content/full/23/10/1704?rss=1')" value="Save"></span>
<span class="itemfrom">[<a href="http://genome.cshlp.org">Genome Research current issue</a>]</span>
</p>
<div class="itemdescription">
<p>Our body habitat-associated microbial communities are of intense research interest because of their influence on human health. Because many studies of the microbiota are based on the same bacterial 16S ribosomal RNA (rRNA) gene target, they can, in principle, be compared to determine the relative importance of different disease/physiologic/developmental states. However, differences in experimental protocols used may produce variation that outweighs biological differences. By comparing 16S rRNA gene sequences generated from diverse studies of the human microbiota using the QIIME database, we found that variation in composition of the microbiota across different body sites was consistently larger than technical variability across studies. However, samples from different studies of the Western adult fecal microbiota generally clustered by study, and the 16S rRNA target region, DNA extraction technique, and sequencing platform produced systematic biases in observed diversity that could obscure biologically meaningful compositional differences. In contrast, systematic compositional differences in the fecal microbiota that occurred with age and between Western and more agrarian cultures were great enough to outweigh technical variation. Furthermore, individuals with ileal Crohn's disease and in their third trimester of pregnancy often resembled infants from different studies more than controls from the same study, indicating parallel compositional attributes of these distinct developmental/physiological/disease states. Together, these results show that cross-study comparisons of human microbiota are valuable when the studied parameter has a large effect size, but studies of more subtle effects on the human microbiota require carefully selected control populations and standardized protocols.</p>
</div>
</div>
<div class="item feed-f637ce59 feed-genomeresearchcurrentissue" id="item-41ff2091">
<p class="itemheader">
<span class="itemtitle"><a href="http://genome.cshlp.org/cgi/content/full/23/10/1690?rss=1">DNA damage induces targeted, genome-wide variation of poly(A) sites in budding yeast [RESEARCH]</a></span>
<span class="itemsave"><input type="button" onclick="sendPatch('http://genome.cshlp.org/cgi/content/full/23/10/1690?rss=1')" value="Save"></span>
<span class="itemfrom">[<a href="http://genome.cshlp.org">Genome Research current issue</a>]</span>
</p>
<div class="itemdescription">
<p>Systemic response to DNA damage and other stresses is a complex process that includes changes in the regulation and activity of nearly all stages of gene expression. One gene regulatory mechanism used by eukaryotes is selection among alternative transcript isoforms that differ in polyadenylation [poly(A)] sites, resulting in changes either to the coding sequence or to portions of the 3' UTR that govern translation, stability, and localization. To determine the extent to which this means of regulation is used in response to DNA damage, we conducted a global analysis of poly(A) site usage in <i>Saccharomyces cerevisiae</i> after exposure to the UV mimetic, 4-nitroquinoline 1-oxide (4NQO). Two thousand thirty-one genes were found to have significant variation in poly(A) site distributions following 4NQO treatment, with a strong bias toward loss of short transcripts, including many with poly(A) sites located within the protein coding sequence (CDS). We further explored one possible mechanism that could contribute to the widespread differences in mRNA isoforms. The change in poly(A) site profile was associated with an inhibition of cleavage and polyadenylation in cell extract and a decrease in the levels of several key subunits in the mRNA 3'-end processing complex. Sequence analysis identified differences in the <i>cis</i>-acting elements that flank putatively suppressed and enhanced poly(A) sites, suggesting a mechanism that could discriminate between variable and constitutive poly(A) sites. Our analysis indicates that variation in mRNA length is an important part of the regulatory response to DNA damage.</p>
</div>
</div>
<div class="item feed-f637ce59 feed-genomeresearchcurrentissue" id="item-65d3bf04">
<p class="itemheader">
<span class="itemtitle"><a href="http://genome.cshlp.org/cgi/content/full/23/10/1675?rss=1">Multiple RNA recognition patterns during microRNA biogenesis in plants [RESEARCH]</a></span>
<span class="itemsave"><input type="button" onclick="sendPatch('http://genome.cshlp.org/cgi/content/full/23/10/1675?rss=1')" value="Save"></span>
<span class="itemfrom">[<a href="http://genome.cshlp.org">Genome Research current issue</a>]</span>
</p>
<div class="itemdescription">
<p>MicroRNAs (miRNAs) derive from longer precursors with fold-back structures. While animal miRNA precursors have homogenous structures, plant precursors comprise a collection of fold-backs with variable size and shape. Here, we design an approach to systematically analyze miRNA processing intermediates and characterize the biogenesis of most of the evolutionarily conserved miRNAs present in <i>Arabidopsis thaliana</i>. We found that plant miRNAs are processed by four mechanisms, depending on the sequential direction of the processing machinery and the number of cuts required to release the miRNA. Classification of the precursors according to their processing mechanism revealed specific structural determinants for each group. We found that the complexity of the miRNA processing pathways occurs in both ancient and evolutionarily young sequences and that members of the same family can be processed in different ways. We observed that different structural determinants compete for the processing machinery and that alternative miRNAs can be generated from a single precursor. The results provide an explanation for the structural diversity of miRNA precursors in plants and new insights toward the understanding of the biogenesis of small RNAs.</p>
</div>
</div>
<div class="item feed-f637ce59 feed-genomeresearchcurrentissue" id="item-78c5ff7b">
<p class="itemheader">
<span class="itemtitle"><a href="http://genome.cshlp.org/cgi/content/full/23/10/1663?rss=1">Epigenome-wide inheritance of cytosine methylation variants in a recombinant inbred population [RESEARCH]</a></span>
<span class="itemsave"><input type="button" onclick="sendPatch('http://genome.cshlp.org/cgi/content/full/23/10/1663?rss=1')" value="Save"></span>
<span class="itemfrom">[<a href="http://genome.cshlp.org">Genome Research current issue</a>]</span>
</p>
<div class="itemdescription">
<p>Cytosine DNA methylation is one avenue for passing information through cell divisions. Here, we present epigenomic analyses of soybean recombinant inbred lines (RILs) and their parents. Identification of differentially methylated regions (DMRs) revealed that DMRs mostly cosegregated with the genotype from which they were derived, but examples of the uncoupling of genotype and epigenotype were identified. Linkage mapping of methylation states assessed from whole-genome bisulfite sequencing of 83 RILs uncovered widespread evidence for local methylQTL. This epigenomics approach provides a comprehensive study of the patterns and heritability of methylation variants in a complex genetic population over multiple generations, paving the way for understanding how methylation variants contribute to phenotypic variation.</p>
</div>
</div>
<div class="item feed-f637ce59 feed-genomeresearchcurrentissue" id="item-95fd5984">
<p class="itemheader">
<span class="itemtitle"><a href="http://genome.cshlp.org/cgi/content/full/23/10/1651?rss=1">The maize methylome influences mRNA splice sites and reveals widespread paramutation-like switches guided by small RNA [RESEARCH]</a></span>
<span class="itemsave"><input type="button" onclick="sendPatch('http://genome.cshlp.org/cgi/content/full/23/10/1651?rss=1')" value="Save"></span>
<span class="itemfrom">[<a href="http://genome.cshlp.org">Genome Research current issue</a>]</span>
</p>
<div class="itemdescription">
<p>The maize genome, with its large complement of transposons and repeats, is a paradigm for the study of epigenetic mechanisms such as paramutation and imprinting. Here, we present the genome-wide map of cytosine methylation for two maize inbred lines, B73 and Mo17. CG (65%) and CHG (50%) methylation (where H = A, C, or T) is highest in transposons, while CHH (5%) methylation is likely guided by 24-nt, but not 21-nt, small interfering RNAs (siRNAs). Correlations with methylation patterns suggest that CG methylation in exons (8%) may deter insertion of <i>Mutator</i> transposon insertion, while CHG methylation at splice acceptor sites may inhibit RNA splicing. Using the methylation map as a guide, we used low-coverage sequencing to show that parental methylation differences are inherited by recombinant inbred lines. However, frequent methylation switches, guided by siRNA, persist for up to eight generations, suggesting that epigenetic inheritance resembling paramutation is much more common than previously supposed. The methylation map will provide an invaluable resource for epigenetic studies in maize.</p>
</div>
</div>
<div class="item feed-f637ce59 feed-genomeresearchcurrentissue" id="item-6bb4cc41">
<p class="itemheader">
<span class="itemtitle"><a href="http://genome.cshlp.org/cgi/content/full/23/10/1636?rss=1">Global analyses of UPF1 binding and function reveal expanded scope of nonsense-mediated mRNA decay [RESEARCH]</a></span>
<span class="itemsave"><input type="button" onclick="sendPatch('http://genome.cshlp.org/cgi/content/full/23/10/1636?rss=1')" value="Save"></span>
<span class="itemfrom">[<a href="http://genome.cshlp.org">Genome Research current issue</a>]</span>
</p>
<div class="itemdescription">
<p>UPF1 is a DNA/RNA helicase with essential roles in nonsense-mediated mRNA decay (NMD) and embryonic development. How UPF1 regulates target abundance and the relationship between NMD and embryogenesis are not well understood. To explore how NMD shapes the embryonic transcriptome, we integrated genome-wide analyses of UPF1 binding locations, NMD-regulated gene expression, and translation in murine embryonic stem cells (mESCs). We identified over 200 direct UPF1 binding targets using crosslinking/immunoprecipitation-sequencing (CLIP-seq) and revealed a repression pathway that involves 3' UTR binding by UPF1 and translation but is independent of canonical targeting features involving 3' UTR length and stop codon placement. Interestingly, NMD targeting of this set of mRNAs occurs in other mouse tissues and is conserved in human. We also show, using ribosome footprint profiling, that actively translated upstream open reading frames (uORFs) are enriched in transcription factor mRNAs and predict mRNA repression by NMD, while poorly translated mRNAs escape repression. Together, our results identify novel NMD determinants and targets and provide context for understanding the impact of UPF1 and NMD on the mESC transcriptome.</p>
</div>
</div>
<div class="item feed-f637ce59 feed-genomeresearchcurrentissue" id="item-71862e09">
<p class="itemheader">
<span class="itemtitle"><a href="http://genome.cshlp.org/cgi/content/full/23/10/1624?rss=1">Genome-wide and parental allele-specific analysis of CTCF and cohesin DNA binding in mouse brain reveals a tissue-specific binding pattern and an association with imprinted differentially methylated regions [RESEARCH]</a></span>
<span class="itemsave"><input type="button" onclick="sendPatch('http://genome.cshlp.org/cgi/content/full/23/10/1624?rss=1')" value="Save"></span>
<span class="itemfrom">[<a href="http://genome.cshlp.org">Genome Research current issue</a>]</span>
</p>
<div class="itemdescription">
<p>DNA binding factors are essential for regulating gene expression. CTCF and cohesin are DNA binding factors with central roles in chromatin organization and gene expression. We determined the sites of CTCF and cohesin binding to DNA in mouse brain, genome wide and in an allele-specific manner with high read-depth ChIP-seq. By comparing our results with existing data for mouse liver and embryonic stem (ES) cells, we investigated the tissue specificity of CTCF binding sites. ES cells have fewer unique CTCF binding sites occupied than liver and brain, consistent with a ground-state pattern of CTCF binding that is elaborated during differentiation. CTCF binding sites without the canonical consensus motif were highly tissue specific. In brain, a third of CTCF and cohesin binding sites coincide, consistent with the potential for many interactions between cohesin and CTCF but also many instances of independent action. In the context of genomic imprinting, CTCF and/or cohesin bind to a majority but not all differentially methylated regions, with preferential binding to the unmethylated parental allele. Whether the parental allele-specific methylation was established in the parental germlines or post-fertilization in the embryo is not a determinant in CTCF or cohesin binding. These findings link CTCF and cohesin with the control regions of a subset of imprinted genes, supporting the notion that imprinting control is mechanistically diverse.</p>
</div>
</div>
<div class="item feed-f637ce59 feed-genomeresearchcurrentissue" id="item-610e3d14">
<p class="itemheader">
<span class="itemtitle"><a href="http://genome.cshlp.org/cgi/content/full/23/10/1615?rss=1">Frac-seq reveals isoform-specific recruitment to polyribosomes [RESEARCH]</a></span>
<span class="itemsave"><input type="button" onclick="sendPatch('http://genome.cshlp.org/cgi/content/full/23/10/1615?rss=1')" value="Save"></span>
<span class="itemfrom">[<a href="http://genome.cshlp.org">Genome Research current issue</a>]</span>
</p>
<div class="itemdescription">
<p>Pre-mRNA splicing is required for the accurate expression of virtually all human protein coding genes. However, splicing also plays important roles in coordinating subsequent steps of pre-mRNA processing such as polyadenylation and mRNA export. Here, we test the hypothesis that nuclear pre-mRNA processing influences the polyribosome association of alternative mRNA isoforms. By comparing isoform ratios in cytoplasmic and polyribosomal extracts, we determined that the alternative products of ~30% (597/1954) of mRNA processing events are differentially partitioned between these subcellular fractions. Many of the events exhibiting isoform-specific polyribosome association are highly conserved across mammalian genomes, underscoring their possible biological importance. We find that differences in polyribosome association may be explained, at least in part by the observation that alternative splicing alters the <i>cis</i>-regulatory landscape of mRNAs isoforms. For example, inclusion or exclusion of upstream open reading frames (uORFs) in the 5'UTR as well as <i>Alu</i>-elements and microRNA target sites in the 3'UTR have a strong influence on polyribosome association of alternative mRNA isoforms. Taken together, our data demonstrate for the first time the potential link between alternative splicing and translational control of the resultant mRNA isoforms.</p>
</div>
</div>
<div class="item feed-f637ce59 feed-genomeresearchcurrentissue" id="item-fe9d8827">
<p class="itemheader">
<span class="itemtitle"><a href="http://genome.cshlp.org/cgi/content/full/23/10/1601?rss=1">De novo DNA demethylation and noncoding transcription define active intergenic regulatory elements [RESEARCH]</a></span>
<span class="itemsave"><input type="button" onclick="sendPatch('http://genome.cshlp.org/cgi/content/full/23/10/1601?rss=1')" value="Save"></span>
<span class="itemfrom">[<a href="http://genome.cshlp.org">Genome Research current issue</a>]</span>
</p>
<div class="itemdescription">
<p>Deep sequencing of mammalian DNA methylomes has uncovered a previously unpredicted number of discrete hypomethylated regions in intergenic space (iHMRs). Here, we combined whole-genome bisulfite sequencing data with extensive gene expression and chromatin-state data to define functional classes of iHMRs, and to reconstruct the dynamics of their establishment in a developmental setting. Comparing HMR profiles in embryonic stem and primary blood cells, we show that iHMRs mark an exclusive subset of active DNase hypersensitive sites (DHS), and that both developmentally constitutive and cell-type-specific iHMRs display chromatin states typical of distinct regulatory elements. We also observe that iHMR changes are more predictive of nearby gene activity than the promoter HMR itself, and that expression of noncoding RNAs within the iHMR accompanies full activation and complete demethylation of mature B cell enhancers. Conserved sequence features corresponding to iHMR transcript start sites, including a discernible TATA motif, suggest a conserved, functional role for transcription in these regions. Similarly, we explored both primate-specific and human population variation at iHMRs, finding that while enhancer iHMRs are more variable in sequence and methylation status than any other functional class, conservation of the TATA box is highly predictive of iHMR maintenance, reflecting the impact of sequence plasticity and transcriptional signals on iHMR establishment. Overall, our analysis allowed us to construct a three-step timeline in which (1) intergenic DHS are pre-established in the stem cell, (2) partial demethylation of blood-specific intergenic DHSs occurs in blood progenitors, and (3) complete iHMR formation and transcription coincide with enhancer activation in lymphoid-specified cells.</p>
</div>
</div>
<div class="item feed-f637ce59 feed-genomeresearchcurrentissue" id="item-f7c89bd6">
<p class="itemheader">
<span class="itemtitle"><a href="http://genome.cshlp.org/cgi/content/full/23/10/1590?rss=1">GC skew at the 5' and 3' ends of human genes links R-loop formation to epigenetic regulation and transcription termination [RESEARCH]</a></span>
<span class="itemsave"><input type="button" onclick="sendPatch('http://genome.cshlp.org/cgi/content/full/23/10/1590?rss=1')" value="Save"></span>
<span class="itemfrom">[<a href="http://genome.cshlp.org">Genome Research current issue</a>]</span>
</p>
<div class="itemdescription">
<p>Strand asymmetry in the distribution of guanines and cytosines, measured by GC skew, predisposes DNA sequences toward R-loop formation upon transcription. Previous work revealed that GC skew and R-loop formation associate with a core set of unmethylated CpG island (CGI) promoters in the human genome. Here, we show that GC skew can distinguish four classes of promoters, including three types of CGI promoters, each associated with unique epigenetic and gene ontology signatures. In particular, we identify a strong and a weak class of CGI promoters and show that these loci are enriched in distinct chromosomal territories reflecting the intrinsic strength of their protection against DNA methylation. Interestingly, we show that strong CGI promoters are depleted from the X chromosome while weak CGIs are enriched, a property consistent with the acquisition of DNA methylation during dosage compensation. Furthermore, we identify a third class of CGI promoters based on its unique GC skew profile and show that this gene set is enriched for Polycomb group targets. Lastly, we show that nearly 2000 genes harbor GC skew at their 3' ends and that these genes are preferentially located in gene-dense regions and tend to be closely arranged. Genomic profiling of R-loops accordingly showed that a large proportion of genes with terminal GC skew form R-loops at their 3' ends, consistent with a role for these structures in permitting efficient transcription termination. Altogether, we show that GC skew and R-loop formation offer significant insights into the epigenetic regulation, genomic organization, and function of human genes.</p>
</div>
</div>
<div class="item feed-f637ce59 feed-genomeresearchcurrentissue" id="item-f7e73750">
<p class="itemheader">
<span class="itemtitle"><a href="http://genome.cshlp.org/cgi/content/full/23/10/1580?rss=1">Lamin A/C-promoter interactions specify chromatin state-dependent transcription outcomes [RESEARCH]</a></span>
<span class="itemsave"><input type="button" onclick="sendPatch('http://genome.cshlp.org/cgi/content/full/23/10/1580?rss=1')" value="Save"></span>
<span class="itemfrom">[<a href="http://genome.cshlp.org">Genome Research current issue</a>]</span>
</p>
<div class="itemdescription">
<p>The nuclear lamina is implicated in the organization of the eukaryotic nucleus. Association of nuclear lamins with the genome occurs through large chromatin domains including mostly, but not exclusively, repressed genes. How lamin interactions with regulatory elements modulate gene expression in different cellular contexts is unknown. We show here that in human adipose tissue stem cells, lamin A/C interacts with distinct spatially restricted subpromoter regions, both within and outside peripheral and intra-nuclear lamin-rich domains. These localized interactions are associated with distinct transcriptional outcomes in a manner dependent on local chromatin modifications. Down-regulation of lamin A/C leads to dissociation of lamin A/C from promoters and remodels repressive and permissive histone modifications by enhancing transcriptional permissiveness, but is not sufficient to elicit gene activation. Adipogenic differentiation resets a large number of lamin-genome associations globally and at subpromoter levels and redefines associated transcription outputs. We propose that lamin A/C acts as a modulator of local gene expression outcome through interaction with adjustable sites on promoters, and that these position-dependent transcriptional readouts may be reset upon differentiation.</p>
</div>
</div>
<div class="item feed-f637ce59 feed-genomeresearchcurrentissue" id="item-829146a5">
<p class="itemheader">
<span class="itemtitle"><a href="http://genome.cshlp.org/cgi/content/full/23/10/1563?rss=1">Sumoylation at chromatin governs coordinated repression of a transcriptional program essential for cell growth and proliferation [RESEARCH]</a></span>
<span class="itemsave"><input type="button" onclick="sendPatch('http://genome.cshlp.org/cgi/content/full/23/10/1563?rss=1')" value="Save"></span>
<span class="itemfrom">[<a href="http://genome.cshlp.org">Genome Research current issue</a>]</span>
</p>
<div class="itemdescription">
<p>Despite numerous studies on specific sumoylated transcriptional regulators, the global role of SUMO on chromatin in relation to transcription regulation remains largely unknown. Here, we determined the genome-wide localization of SUMO1 and SUMO2/3, as well as of UBC9 (encoded by <i>UBE2I</i>) and PIASY (encoded by <i>PIAS4</i>), two markers for active sumoylation, along with Pol II and histone marks in proliferating versus senescent human fibroblasts together with gene expression profiling. We found that, whereas SUMO alone is widely distributed over the genome with strong association at active promoters, active sumoylation occurs most prominently at promoters of histone and protein biogenesis genes, as well as Pol I rRNAs and Pol III tRNAs. Remarkably, these four classes of genes are up-regulated by inhibition of sumoylation, indicating that SUMO normally acts to restrain their expression. In line with this finding, sumoylation-deficient cells show an increase in both cell size and global protein levels. Strikingly, we found that in senescent cells, the SUMO machinery is selectively retained at histone and tRNA gene clusters, whereas it is massively released from all other unique chromatin regions. These data, which reveal the highly dynamic nature of the SUMO landscape, suggest that maintenance of a repressive environment at histone and tRNA loci is a hallmark of the senescent state. The approach taken in our study thus permitted the identification of a common biological output and uncovered hitherto unknown functions for active sumoylation at chromatin as a key mechanism that, in dynamically marking chromatin by a simple modifier, orchestrates concerted transcriptional regulation of a network of genes essential for cell growth and proliferation.</p>
</div>
</div>
<div class="item feed-f637ce59 feed-genomeresearchcurrentissue" id="item-72eb0c8b">
<p class="itemheader">
<span class="itemtitle"><a href="http://genome.cshlp.org/cgi/content/full/23/10/1749?rss=1">The million mutation project: A new approach to genetics in Caenorhabditis elegans [RESOURCES]</a></span>
<span class="itemsave"><input type="button" onclick="sendPatch('http://genome.cshlp.org/cgi/content/full/23/10/1749?rss=1')" value="Save"></span>
<span class="itemfrom">[<a href="http://genome.cshlp.org">Genome Research current issue</a>]</span>
</p>
<div class="itemdescription">
<p>We have created a library of 2007 mutagenized <i>Caenorhabditis elegans</i> strains, each sequenced to a target depth of 15-fold coverage, to provide the research community with mutant alleles for each of the worm's more than 20,000 genes. The library contains over 800,000 unique single nucleotide variants (SNVs) with an average of eight nonsynonymous changes per gene and more than 16,000 insertion/deletion (indel) and copy number changes, providing an unprecedented genetic resource for this multicellular organism. To supplement this collection, we also sequenced 40 wild isolates, identifying more than 630,000 unique SNVs and 220,000 indels. Comparison of the two sets demonstrates that the mutant collection has a much richer array of both nonsense and missense mutations than the wild isolate set. We also find a wide range of rDNA and telomere repeat copy number in both sets. Scanning the mutant collection for molecular phenotypes reveals a nonsense suppressor as well as strains with higher levels of indels that harbor mutations in DNA repair genes and strains with abundant males associated with <i>him</i> mutations. All the strains are available through the Caenorhabditis Genetics Center and all the sequence changes have been deposited in WormBase and are available through an interactive website.</p>
</div>
</div>
<div class="item feed-f637ce59 feed-genomeresearchcurrentissue" id="item-ad13dff8">
<p class="itemheader">
<span class="itemtitle"><a href="http://genome.cshlp.org/cgi/content/full/23/10/1740?rss=1">Coelacanth genomes reveal signatures for evolutionary transition from water to land [RESOURCES]</a></span>
<span class="itemsave"><input type="button" onclick="sendPatch('http://genome.cshlp.org/cgi/content/full/23/10/1740?rss=1')" value="Save"></span>
<span class="itemfrom">[<a href="http://genome.cshlp.org">Genome Research current issue</a>]</span>
</p>
<div class="itemdescription">
<p>Coelacanths are known as "living fossils," as they show remarkable morphological resemblance to the fossil record and belong to the most primitive lineage of living Sarcopterygii (lobe-finned fishes and tetrapods). Coelacanths may be key to elucidating the tempo and mode of evolution from fish to tetrapods. Here, we report the genome sequences of five coelacanths, including four <i>Latimeria chalumnae</i> individuals (three specimens from Tanzania and one from Comoros) and one <i>L. menadoensis</i> individual from Indonesia. These sequences cover two African breeding populations and two known extant coelacanth species. The genome is ~2.74 Gbp and contains a high proportion (~60%) of repetitive elements. The genetic diversity among the individuals was extremely low, suggesting a small population size and/or a slow rate of evolution. We found a substantial number of genes that encode olfactory and pheromone receptors with features characteristic of tetrapod receptors for the detection of airborne ligands. We also found that limb enhancers of <i>bmp7</i> and <i>gli3</i>, both of which are essential for limb formation, are conserved between coelacanth and tetrapods, but not ray-finned fishes. We expect that some tetrapod-like genes may have existed early in the evolution of primitive Sarcopterygii and were later co-opted to adapt to terrestrial environments. These coelacanth genomes will provide a cornerstone for studies to elucidate how ancestral aquatic vertebrates evolved into terrestrial animals.</p>
</div>
</div>
<div class="item feed-f637ce59 feed-genomeresearchcurrentissue" id="item-9c050fcf">
<p class="itemheader">
<span class="itemtitle"><a href="http://genome.cshlp.org/cgi/content/full/23/10/1730?rss=1">Integrated detection of natural antisense transcripts using strand-specific RNA sequencing data [METHOD]</a></span>
<span class="itemsave"><input type="button" onclick="sendPatch('http://genome.cshlp.org/cgi/content/full/23/10/1730?rss=1')" value="Save"></span>
<span class="itemfrom">[<a href="http://genome.cshlp.org">Genome Research current issue</a>]</span>
</p>
<div class="itemdescription">
<p>Pairs of RNA molecules transcribed from partially or entirely complementary loci are called <i>cis</i>-natural antisense transcripts (<i>cis</i>-NATs), and they play key roles in the regulation of gene expression in many organisms. A promising experimental tool for profiling sense and antisense transcription is strand-specific RNA sequencing (ssRNA-seq). To identify <i>cis</i>-NATs using ssRNA-seq, we developed a new computational method based on a model comparison framework that incorporates the inherent variable efficiency of generating perfectly strand-specific libraries. Applying the method to new ssRNA-seq data from whole-root and cell-type&ndash;specific <i>Arabidopsis</i> libraries confirmed most of the known <i>cis</i>-NAT pairs and identified 918 additional <i>cis</i>-NAT pairs. Newly identified <i>cis</i>-NAT pairs are supported by polyadenylation data, alternative splicing patterns, and RT-PCR validation. We found 209 <i>cis</i>-NAT pairs that have opposite expression levels in neighboring cell types, implying cell-type&ndash;specific roles for <i>cis</i>-NATs. By integrating a genome-wide epigenetic profile of <i>Arabidopsis</i>, we identified a unique chromatin signature of <i>cis</i>-NATs, suggesting a connection between <i>cis</i>-NAT transcription and chromatin modification in plants. An analysis of small-RNA sequencing data showed that ~4% of <i>cis</i>-NAT pairs produce putative <i>cis</i>-NAT&ndash;induced siRNAs. Taken together, our data and analyses illustrate the potential for multifaceted regulatory roles of plant <i>cis</i>-NATs.</p>
</div>
</div>
<div class="item feed-f637ce59 feed-genomeresearchcurrentissue" id="item-b9b43620">
<p class="itemheader">
<span class="itemtitle"><a href="http://genome.cshlp.org/cgi/content/full/23/10/1721?rss=1">Pathoscope: Species identification and strain attribution with unassembled sequencing data [METHOD]</a></span>
<span class="itemsave"><input type="button" onclick="sendPatch('http://genome.cshlp.org/cgi/content/full/23/10/1721?rss=1')" value="Save"></span>
<span class="itemfrom">[<a href="http://genome.cshlp.org">Genome Research current issue</a>]</span>
</p>
<div class="itemdescription">
<p>Emerging next-generation sequencing technologies have revolutionized the collection of genomic data for applications in bioforensics, biosurveillance, and for use in clinical settings. However, to make the most of these new data, new methodology needs to be developed that can accommodate large volumes of genetic data in a computationally efficient manner. We present a statistical framework to analyze raw next-generation sequence reads from purified or mixed environmental or targeted infected tissue samples for rapid species identification and strain attribution against a robust database of known biological agents. Our method, <i>Pathoscope</i>, capitalizes on a Bayesian statistical framework that accommodates information on sequence quality, mapping quality, and provides posterior probabilities of matches to a known database of target genomes. Importantly, our approach also incorporates the possibility that multiple species can be present in the sample and considers cases when the sample species/strain is not in the reference database. Furthermore, our approach can accurately discriminate between very closely related strains of the same species with very little coverage of the genome and without the need for multiple alignment steps, extensive homology searches, or genome assembly&mdash;which are time-consuming and labor-intensive steps. We demonstrate the utility of our approach on genomic data from purified and in silico "environmental" samples from known bacterial agents impacting human health for accuracy assessment and comparison with other approaches.</p>
</div>
</div>
<div class="item feed-f637ce59 feed-genomeresearchcurrentissue" id="item-f2f7fa47">
<p class="itemheader">
<span class="itemtitle"><a href="http://genome.cshlp.org/cgi/content/full/23/10/1715?rss=1">Sympatric chimpanzees and gorillas harbor convergent gut microbial communities [RESEARCH]</a></span>
<span class="itemsave"><input type="button" onclick="sendPatch('http://genome.cshlp.org/cgi/content/full/23/10/1715?rss=1')" value="Save"></span>
<span class="itemfrom">[<a href="http://genome.cshlp.org">Genome Research current issue</a>]</span>
</p>
<div class="itemdescription">
<p>The gut microbial communities within great apes have been shown to reflect the phylogenetic history of their hosts, indicating codiversification between great apes and their gut microbiota over evolutionary timescales. But because the great apes examined to date represent geographically isolated populations whose diets derive from different sources, it is unclear whether this pattern of codiversification has resulted from a long history of coadaptation between microbes and hosts (heritable factors) or from the ecological and geographic separation among host species (environmental factors). To evaluate the relative influences of heritable and environmental factors on the evolution of the great ape gut microbiota, we assayed the gut communities of sympatric and allopatric populations of chimpanzees, bonobos, and gorillas residing throughout equatorial Africa. Comparisons of these populations revealed that the gut communities of different host species can always be distinguished from one another but that the gut communities of sympatric chimpanzees and gorillas have converged in terms of community composition, sharing on average 53% more bacterial phylotypes than the gut communities of allopatric hosts. Host environment, independent of host genetics and evolutionary history, shaped the distribution of bacterial phylotypes across the Bacteroidetes, Firmicutes, Proteobacteria, and Actinobacteria, the four most common phyla of gut bacteria. Moreover, the specific patterns of phylotype sharing among hosts suggest that chimpanzees living in sympatry with gorillas have acquired bacteria from gorillas. These results indicate that geographic isolation between host species has promoted the evolutionary differentiation of great ape gut bacterial communities.</p>
</div>
</div>
</ol></div>

<div class="feeddisplay">
<h3 class="feedtitle"><a id="2"><a href="http://www.biodatamining.org/">BioData Mining - Latest Articles</a></a></h3>
<a href="#1">prev</a> | <a href="#3">next</a>
<ol class="feedarticles">
<div class="item feed-0320d9ef feed-biodatamininglatestarticles" id="item-2a98c5bc">
<p class="itemheader">
<span class="itemtitle"><a href="http://www.biodatamining.org/content/6/1/16">LVQ-SMOTE &#191;Learning Vector Quantization based Synthetic Minority Over&#191;sampling Technique for biomedical data</a></span>
<span class="itemsave"><input type="button" onclick="sendPatch('http://www.biodatamining.org/content/6/1/16')" value="Save"></span>
<span class="itemfrom">[<a href="http://www.biodatamining.org/">BioData Mining - Latest Articles</a>]</span>
</p>
<div class="itemdescription">
<p>Background:
Over-sampling methods based on Synthetic Minority Over-sampling Technique (SMOTE) have beenproposed for classification problems of imbalanced biomedical data. However, the existing oversamplingmethods achieve slightly better or sometimes worse result than the simplest SMOTE. In orderto improve the effectiveness of SMOTE, this paper presents a novel over-sampling method usingcodebooks obtained by the learning vector quantization. In general, even when an existing SMOTEapplied to a biomedical dataset, its empty feature space is still so huge that most classification algorithmswould not perform well on estimating borderlines between classes. To tackle this problem, ourover-sampling method generates synthetic samples which occupy more feature space than the otherSMOTE algorithms. Briefly saying, our over-sampling method enables to generate useful syntheticsamples by referring to actual samples taken from real-world datasets.
Results:
Experiments on eight real-world imbalanced datasets demonstrate that our proposed over-samplingmethod performs better than the simplest SMOTE on four of five standard classification algorithms.Moreover, it is seen that the performance of our method increases if the latest SMOTE called MWMOTEis used in our algorithm. Experiments on datasets for &#223;-turn types prediction show someimportant patterns that have not been seen in previous analyses.
Conclusions:
The proposed over-sampling method generates useful synthetic samples for the classification of imbalancedbiomedical data. Besides, the proposed over-sampling method is basically compatible withbasic classification algorithms and the existing over-sampling methods.
</div>
</div>
</ol></div>
<a id="3"><a href="#2">prev</a></a>
</div>
<h2 id="feedstatsheader">Feeds</h2>
<div id="feedstats">
<table id="feeds">
<tr id="feedsheader">
<th>Feed</th><th>RSS</th><th>Last fetched</th><th>Next fetched after</th>
</tr>

<tr class="feedsrow">
<td><a href="http://www.biodatamining.org/">BioData Mining - Latest Articles</a></td>
<td><a class="xmlbutton" href="http://www.biodatamining.org/latest/rss">XML</a></td>
<td>2013-10-04 10:11</td>
<td>2013-10-05 10:11</td>
</tr>

<tr class="feedsrow">
<td><a href="http://genome.cshlp.org">Genome Research current issue</a></td>
<td><a class="xmlbutton" href="http://genome.cshlp.org/rss/current.xml">XML</a></td>
<td>2013-10-04 10:11</td>
<td>2013-10-05 10:11</td>
</tr>

</table>

</div>
<div id="footer">
<p id="aboutrawdog">Generated by
<a href="http://offog.org/code/rawdog.html">rawdog</a>
version 2.18
by <a href="mailto:ats@offog.org">Adam Sampson</a>.</p>
</div>
</body>
</html>
